.github/workflows/scripts/create_pr.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -eo pipefail
+
+VERSION=$1
+GITHUB_EVENT_NAME=$2
+BRANCH_NAME="update-changelog-$VERSION-$(date +%F_%H-%M-%S)"
+
+(
+  cd ./loadgen/cmd/otelbench
```

Comment:
Actually I think I made a mistake before. So this script is called from the GH workflow, so we are at the root. Then we use `cd` here, so when the scope ends, we are back to the root directory. So I removed pushd and popd. Is this correct @v1v ?